src/textual/app.py

```diff
@@ -754,6 +762,28 @@ async def process_messages(self) -> None:
                 self._log_file.close()
                 self._log_console = None
 
+    async def _ready(self) -> None:
+        """Called immediately prior to processing messages.
+
+        May be used as a hook for any operations that should run first.
+
+        """
+        try:
+            screenshot_timer = float(os.environ.get("TEXTUAL_SCREENSHOT", "0"))
```

Comment:
I know this is made at build time and we're only waiting for 0.1 second which is quite short, but we may even use a fake clock to make it even faster if we want to, now that we have this machinery in place :slightly_smiling_face:

Reply:
I'll revisit that at some point. I think we could avoid the delay entirely. If we have dozens of these, it might add up!